app/src/main/java/edu/uc/amplify/MainActivity.kt

```diff
@@ -43,7 +43,7 @@ class MainActivity : AppCompatActivity() {
                     // Now you can start interacting with App Remote
                     connected()
                 }
-
+                // This function will throw an error message on failure.
```

Comment:
This comment:
1) Is not needed, since it is an overridden function.  Refer to the superclass/interface doucmentation.
2) If here, should be Javadoc format, not inline format.